Manage Plugins not working 1.5.1 (wxPython3)

When we select "Manage Plugins" the following errors are printed:
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/robotide/ui/actiontriggers.py", line 377, in __call__
    action.act(event)
  File "/usr/lib/python2.7/site-packages/robotide/action/action.py", line 86, in act
    self.action(event)
  File "/usr/lib/python2.7/site-packages/robotide/utils/eventhandler.py", line 24, in wrapped
    method(self, event)
  File "/usr/lib/python2.7/site-packages/robotide/ui/mainframe.py", line 253, in OnManagePlugins
    self._plugin_manager.show(self._application.get_plugins())
  File "/usr/lib/python2.7/site-packages/robotide/ui/pluginmanager.py", line 31, in show
    self._tab = _PluginPanel(self._notebook, plugins, self._show_tab)
  File "/usr/lib/python2.7/site-packages/robotide/ui/pluginmanager.py", line 47, in __init__
    sizer.Add(self._create_body(plugins, activation_callback), 1, flag=wx.EXPAND|wx.ALL, border=16)
  File "/usr/lib/python2.7/site-packages/robotide/ui/pluginmanager.py", line 69, in _create_body
    flag=wx.ALIGN_CENTER_HORIZONTAL)
  File "/usr/lib64/python2.7/site-packages/wx-3.0-gtk3/wx/_core.py", line 14453, in Add
    return _core_.Sizer_Add(*args, **kwargs)
wx._core.PyAssertionError: C++ assertion "Assert failure" failed at ./src/common/sizer.cpp(1401) in DoInsert(): too many items (3 > 2*1) in grid sizer (maybe you should omit the number of either rows or columns?)
```

--- a/src/robotide/ui/pluginmanager.py
+++ b/src/robotide/ui/pluginmanager.py
@@ -29,7 +29,8 @@ class PluginManager(object):
     def show(self, plugins):
         if not self._tab:
             self._tab = _PluginPanel(self._notebook, plugins, self._show_tab)
-            self._notebook.add_tab(self._tab, 'Manage Plugins', allow_closing=true)
+            self._notebook.add_tab(self._tab, 'Manage Plugins',
+                                   allow_closing=True)
         self._show_tab()
 
     def _show_tab(self):
@@ -41,10 +42,13 @@ class _PluginPanel(wx.Panel):
     def __init__(self, notebook, plugins, activation_callback):
         wx.Panel.__init__(self, notebook)
         sizer = wx.BoxSizer(wx.VERTICAL)
-        sizer.Add(self._create_header(), 0, flag=wx.LEFT|wx.RIGHT|wx.TOP, border=16)
-        sizer.Add(self._create_info_text(), 0, flag=wx.EXPAND|wx.LEFT|wx.RIGHT, border=16)
-        sizer.Add(self._create_line(), 0, flag=wx.EXPAND|wx.LEFT|wx.RIGHT)
-        sizer.Add(self._create_body(plugins, activation_callback), 1, flag=wx.EXPAND|wx.ALL, border=16)
+        sizer.Add(self._create_header(), 0, flag=wx.LEFT | wx.RIGHT | wx.TOP,
+                  border=16)
+        sizer.Add(self._create_info_text(), 0,
+                  flag=wx.EXPAND | wx.LEFT | wx.RIGHT, border=16)
+        sizer.Add(self._create_line(), 0, flag=wx.EXPAND | wx.LEFT | wx.RIGHT)
+        sizer.Add(self._create_body(plugins, activation_callback), 1,
+                  flag=wx.EXPAND | wx.ALL, border=16)
         self.SetSizer(sizer)
 
     def _create_header(self):
@@ -59,20 +63,23 @@ class _PluginPanel(wx.Panel):
     def _create_body(self, plugins, activation_callback):
         panel = ScrolledPanel(self, wx.ID_ANY, style=wx.TAB_TRAVERSAL)
         panel.SetupScrolling()
-        sizer = wx.FlexGridSizer(1, 2, hgap=8, vgap=8)
+        sizer = wx.FlexGridSizer(0, 2, hgap=8, vgap=8)
         sizer.AddGrowableCol(1, 1)
         sizer.Add(self._create_label(panel, 'Enabled'), 0, wx.BOTTOM, border=8)
-        sizer.Add(self._create_label(panel, 'Plugin'), 0,
-                  wx.BOTTOM|wx.EXPAND, border=8)
+        sizer.Add(self._create_label(panel, 'Plugin'), 0, wx.BOTTOM |
+                  wx.EXPAND, border=8)
         for plugin in sorted(plugins, key=lambda p: p.name):
-            sizer.Add(_PluginEnablationCheckBox(panel, plugin, activation_callback),
+            sizer.Add(_PluginEnablationCheckBox(panel, plugin,
+                                                activation_callback),
                       flag=wx.ALIGN_CENTER_HORIZONTAL)
             sizer.Add(_PluginRow(panel, plugin), 0, wx.EXPAND)
         panel.SetSizer(sizer)
         return panel
 
     def _create_info_text(self):
-        info = wx.StaticText(self, wx.ID_ANY, "Info. Enabling and disabling plugins might require RIDE restart for menus to work.")
+        info = wx.StaticText(self, wx.ID_ANY,
+                             "Info. Enabling and disabling plugins might \
+require RIDE restart for menus to work.")
         info.SetFont(wx.Font(12, wx.SWISS, wx.NORMAL, wx.FONTWEIGHT_NORMAL))
         return info
 
@@ -108,12 +115,13 @@ class _PluginEnablationCheckBox(wx.CheckBox):
         except Exception, err:
             self.SetValue(False)
             self.Enable(False)
-            msg = 'Failed to %s plugin %s:\n%s'\
-                   % (method.__name__, self._plugin.name, err)
+            msg = 'Failed to %s plugin %s:\n%s' % (method.__name__,
+                                                   self._plugin.name, err)
             self._plugin.error = err
             self._plugin.doc = msg
             LOG.error(msg)
-            RideLogException(message=msg, exception=err, level='ERROR').publish()
+            RideLogException(message=msg, exception=err,
+                             level='ERROR').publish()
 
 
 class _PluginRow(wx.Panel):
@@ -127,7 +135,7 @@ class _PluginRow(wx.Panel):
         sizer.Add(self._get_description(plugin), 0, wx.EXPAND)
         config = plugin.config_panel(self)
         if config:
-            sizer.Add(config, 1, wx.EXPAND|wx.LEFT, border=16)
+            sizer.Add(config, 1, wx.EXPAND | wx.LEFT, border=16)
         self.SetSizer(sizer)
 
     def _get_name(self, plugin):

